src/content/docs/cypher/data-definition/drop.md

```diff
@@ -34,3 +34,13 @@ DROP TABLE User
 | NodeTable: User has been dropped. |
 -------------------------------------
 ```
+
+## IF EXISTS
+By default, kuzu throws an exception if the given table does not exist in the database. `IF EXISTS` syntax instructs kuzu do nothing when
```

Comment:
```suggestion
If the given table does not exist in the database, Kùzu throws an exception when you try to drop it. To avoid the exception being raised, use the `IF EXISTS` clause. This instructs Kùzu to do nothing when
```